test/onnx/test_fx_to_onnx_with_onnxruntime.py

```diff
@@ -244,6 +367,94 @@ def func(x, b=torch.tensor(1.0)):
             self, func, (tensor_x,), b=torch.tensor(5.0)
         )
 
+    # beartype.roar.BeartypeCallHintParamViolation:
+    # @beartyped onnxscript.function_libs.torch_aten.graph_building.TorchScriptGraph.add_input()
+    # parameter input_value=8.0 violates type hint typing.Union[torch.Tensor, NoneType],
+    # as float 8.0 not <class "builtins.NoneType"> or <protocol "torch.Tensor">.
```

Comment:
Hmm. Should we wrap it as a tensor?

Reply:
I thought in general this is the problem of having number type, instead of tensor type, as input for onnx. There is not fundamentally blocking, just not covered yet.

BTW how is 5.0 different than 8.0? Did you mean `tensor(5.0)`?

Reply:
Oh NVM. I was confused by the previous test case.

Reply:
> Hmm. Should we wrap it as a tensor?

I think so. We expects shape/type from inputs.

Reply:
Could we do something like `is_builtin_type(input_value)` and proceed with either its tensor.dtype or scalar type (built-in type)

Although unusual, builtin type is present in models; `bool` should be the most common, but users are free to input other than `Union[torch.Tensor, NoneType]`

Reply:
> Hmm. Should we wrap it as a tensor?

I think this may have side effects. That is why Meta created `SymInt` and `SymFloat`. wrapping as tensor may change the execution path

Reply:
I think the error message seems outdated, as the add_input API has been changed for a while. For SymInt and SymFloat, I think it should be fine if we wrap them as tensor, as we are already doing it in dynamic exporting to enable their calculations(add, multiple, divide, ...). Looking forard to playing around with this test case.

Reply:
Updated error message. Follow up on builtin types separately.